yt/yt/tests/integration/queues/test_queue_agent.py

```diff
@@ -3054,6 +3054,146 @@ def test_export_ttl(self):
         self.remove_export_destination(export_dir)
 
 
+class TestAutomaticTrimmingWithExports(TestQueueStaticExportBase):
+    DELTA_QUEUE_AGENT_DYNAMIC_CONFIG = {
+        "queue_agent": {
+            "controller": {
+                "pass_period": 75,
+                "enable_queue_static_export": True,
+                "enable_automatic_trimming": True,
+            },
+        },
+        "cypress_synchronizer": {
+            "policy": "watching",
+        },
+    }
+
+    @authors("apachee")
+    @pytest.mark.timeout(120)
+    def test_basic(self):
+        queue_agent_orchid = QueueAgentOrchid()
+
+        _, queue_id = self._create_queue("//tmp/q", partition_count=2)
+        export_dir = "//tmp/export"
+        self._create_export_destination(export_dir, queue_id)
+
+        set("//tmp/q/@auto_trim_config", {"enable": True})
+        set("//tmp/q/@static_export_config", {
+            "default": {
+                "export_directory": export_dir,
+                "export_period": 15 * 1000,
```

Comment:
Do we actually need 15 second periods in these tests?
Can we make them 3-4-5 seconds?

All our tests are already incredibly long, you should keep such manual times as low as possible without making flaky tests.